mne/io/eeglab/eeglab.py

```diff
@@ -733,7 +737,31 @@ def _bunchify(items):
     return items
 
 
-def _read_annotations_eeglab(eeg):
+def _read_annotations_eeglab(eeg, output_type=None):
+    r"""Create Annotations from EEGLAB file.
+
+    This function reads the event attribute from the EEGLAB
+    structure and makes an :class:`mne.Annotations` object.
+
+    Parameters
+    ----------
+    eeg : object
+        'EEG' struct
+
+    output_type: str
+        The only valid values are: 'sample' or 'time_stamp'. This parameter is
+        used for back compatibility to replicate a buggy behavior. When
+        output_type='sample' the returned annotation object is not valid since
+        annotation.onset are sample ids rather than time stamps (and this is a
+        BUG). output_type='time_stamp' produce proper annotation object.
```

Comment:
if that was a bug, do we need to keep it as an option?

Reply:
to have backward compatibility until it's deprecated. Maybe its not clear enough on the docstring.